styles/globals.css

```diff
@@ -14,3 +20,20 @@ a {
 * {
   box-sizing: border-box;
 }
+
+h1 {
+  font-size: 100px;
+}
+
+.about-us-caption-button {
```

Comment:
We should not have to use CSS anywhere. We should review https://chakra-ui.com/docs/styled-system/theming/customize-theme and redo this. We can customize specific styling for a chakra Button instead of doing this and using flexbox to position appropriately. The point of chakra is to use their styling conventions! We definitely do not want to flood global.css with styles either.